### DISCUSSION TITLE
Videosorveglianza : consigli per gli acquisti.

### DISCUSSION THREAD

**Discussion by Bergoglio** [indomus/forum#2970] [GENERALE - Discussioni generiche]
Ciao a tutti,

sto pianificando il sistema di videosorveglianza per casa mia e non riesco a decidere alcuni componenti.
Utilizzerò come cam esterne quelle della Annke, le cam all'interno devo ancora sceglierle mentre lato software opterei per Frigate.
Dato che le cam esterne saranno  7 od 8, che PC mi consigliate per far girare Frigate (comprensivo di schede di calcolo Google Coral) ?

Grazie

**Comment by cecc83** [comment]
Ciao, io uso un minipc N95 con 16gb di RAM (la differenza con l'8 era poca cosa in termini di prezzo) e attualmente ho collegato 5 annke AC800 con Frigate e Coral USB e questo è l'utilizzo del processore:

![image](https://github.com/indomus/forum/assets/2249697/332a7f34-2b41-43b6-ad74-c4bac449394a)

la memoria invece ha picchi massimi del 35%, adesso ad esempio è a 4.18GB.
Siccome devo fare un altro impianto identico stavo valutando di prendere l'N100 (che dovrebbe consumare meno), ma comunque credo siano molto simili come performance.

**Comment by Bergoglio** [comment]
Innanzi tutto grazie per la risposta molto esaustiva !!!!

Adesso riesco ad avere una base di partenza per le valutazioni.
Curiosità : salvi le riprese su qualche NAS oppure usi un hard disk interno ?

**Reply by cecc83** [reply]
attualmente il sistema è in fase di test, devo ancora trasferirmi nella casa nuova. Quindi salvo al momento sull'hard disk interno del minipc gli eventi degli ultimi 2 giorni.

**Comment by Mic87** [comment]
Scusate se mi unisco, stavo valutando anche io un mini pc N100 Vorrei un paio di suggerimenti:

1. potrei integrare la scheda Coral pcie togliendo il modulo wifi/bluetooth per ottimizzare consumi e prestazioni?
2. è fattibile eseguire le registrazioni di 4/6 telecamere direttamente su NAS?
3. Lato SW, come strutturate il tutto? Debian-Docker-HA Core, ma per Frigate? Installazione su Docker e poi integrazione con HA o Add-on direttamente in HA?

Grazie a tutti!

**Reply by cecc83** [reply]
Ti rispondo direttamente solo per il punto 3, visto che ho il Coral USB. Se scegli HA Core in Docker non hai gli addon, quindi necessariamente Frigate su Docker, poi usi l'integrazione solo per far "vedere" Frigate ad Home assistant. Io ce l'ho così e mi trovo molto bene, alla fine anche per aggiornare quando esce la nuova versione sono 3 comandi in croce (uso docker compose).
Per il punto 1 stavo valutando anch'io di mettere la versione M2 o pcie ma non so quanto ne vale la pena, adesso sta consumando 12W. Oltretutto la versione USB è plug-n-play mentre le altre hanno bisogno del driver.

**Reply by Mic87** [reply]
Perfetto, intanto lato SW so come procedere. 
Il problema attuale del modello USB è la disponibilità (almeno ad un prezzo umano).
Lato Debian (a parte i driver per la versione pcie o M2), vanno installati Edge TPU e PyCoral? scusa le domande ma non ho mai lavorato con Coral.
Per il discorso modulo pcie, avere la possibilità di inserirlo all'interno del case mi fa stare più tranquillo (nonostante non sia facilmente accessibile, eviterei qualsiasi possibile urto), inoltre proverei a togliere il modulo radio che comunque non userei.

**Reply by cecc83** [reply]
Tranquillo, nessun problema. No, non va installato nulla, totalmente plug-n-play.

**Comment by faustozago** [comment]
Scusate se mi intrometto...ma è possibile usare il modulo CORAL PCIE nell'alloggiamento del modulo WIFI/Bluetooth?
Io pensavo fosse un alloggiamento specifico per quel determinato modulo......(ho un HP prodesk mini , l'alloggiamento M2/Nvme è gia' occupato, dispositivi USB ne ho gia' troppi , in piu' HASSIO gira su VBox .....)

**Reply by Mic87** [reply]
Non conosco il PC descritto, ma teoricamente, se è una porta pcie standard, non ci dovrebbero essere problemi ad utilizzarla per qualsiasi device (a meno di limitazioni lato BIOS)

**Reply by faustozago** [reply]
Perfetto, Grazie
